process/smartContract/process.go

```diff
@@ -927,30 +914,37 @@ func (sc *scProcessor) createSmartContractResult(
 	return result
 }
 
+func (sc *scProcessor) computeActualConsumedGas(
+	tx data.TransactionHandler,
+	vmOutput *vmcommon.VMOutput,
+	acntSnd state.UserAccountHandler,
+) uint64 {
+	consumedGas := tx.GetGasLimit() - vmOutput.GasRemaining
+	if check.IfNil(acntSnd) {
+		consumedGas = consumedGas - sc.economicsFee.ComputeGasLimit(tx)
```

Comment:
+= instead -= ?

Reply:
it is -=

Reply:
So it refers to what is consumed in destination shard only?

Reply:
yes